openassessment/xblock/static/js/spec/lms/oa_response.js

```diff
@@ -23,7 +23,10 @@ describe("OpenAssessment.ResponseView", function() {
     var FILE_TYPE_WHITE_LIST = ['pdf', 'doc', 'docx', 'html'];
     var FILE_EXT_BLACK_LIST = ['exe', 'msi', 'app', 'dmg'];
     var MAXIMUM_FILE_UPLOAD_COUNT = 20;
-    var COURSE_ID = 'course_id'
+    var COURSE_ID = 'course_id';
+
+    var BAD_FILETYPE_MESSAGE = 'File upload failed: unsupported file type. ' +
+        'Only the supported file types can be uploaded. If you have questions, please reach out to the course team.';
```

Comment:
Nit: Perhaps remove `If you have questions, please reach out to the course team.` ?

Reply:
Planning to leave it, since it was provided by UX/Product.